node/src/reactor/joiner.rs

```diff
@@ -630,15 +631,9 @@ impl reactor::Reactor for Reactor {
                             }
                         };
 
-                    let event = match block_at_height {
-                        BlockByHeight::Absent(block_height) => fetcher::Event::AbsentRemotely {
-                            id: block_height,
-                            peer: sender,
-                        },
```

Comment:
How does the peer respond when it does not have the requested block? I.e. what's the replacement of `BlockByHeight::Absent` now?